bin/generate_schema

```diff
@@ -20,6 +20,7 @@ src/manage.py generate_swagger \
 echo "Converting Swagger to OpenAPI 3.0..."
 npm run convert
 patch_content_types
+use_external_components
```

Comment:
Can we generate 2 schema files?

One called `openapi.yaml` which include everything (with internal refs)
One called `openapi_unresolved.yaml` which includes via external refs from `common.yaml`

Reply:
`openapi_unresolved.yaml` is generated with refs to `common.yaml` url